### PR TITLE
add app lab as recommendation on course 4 congrats page

### PR DESCRIPTION
**Pull request by tanyaparker**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
oh haha. I tried to demo this and I didn't merge it in yet. @breville all good?

File: pegasus/sites.v3/code.org/views/course_block.haml [hunk 1]
```diff
@@ -1,4 +1,9 @@
-- url = CDO.studio_url("/s/#{id}")
+- if id.eql? "applab"
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/course_block.haml]: do we need eql? here?

**Comment by breville** [comment]
just one question, otherwise lgtm

File: pegasus/sites.v3/code.org/views/course_block.haml (hunk 1 above)
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/course_block.haml]: Is it better as == ?

Comment (breville) [inline pegasus/sites.v3/code.org/views/course_block.haml]: I think == does the job. 

http://stackoverflow.com/questions/7156955/whats-the-difference-between-equal-eql-and

Comment (tanyaparker) [inline pegasus/sites.v3/code.org/views/course_block.haml]: Oh. Dandy! Will update.

**Pull request merged**